docs/features/authentication.rst

```diff
@@ -176,13 +176,13 @@ Add the following, at minimum, to your startup ``ConfigureServices`` method:
         });
     services.AddOcelot(configuration);
 
-**Note** In order to get Ocelot to view the scope claim from Okta properly, you have to add the following to map the default Okta ``"scp"`` claim to ``"scope"``:
+**Note** In order to get Ocelot to view the scope claim from Okta properly, you have to add the following to map the default Okta ``"scp"`` claim to ``"scope"``[#f4]_:
 
 .. code-block:: csharp
 
     // Map Okta "scp" to "scope" claims instead of http://schemas.microsoft.com/identity/claims/scope to allow Ocelot to read/verify them
-    JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Remove("scp");
-    JwtSecurityTokenHandler.DefaultInboundClaimTypeMap.Add("scp", "scope");
+    JsonWebTokenHandler.DefaultInboundClaimTypeMap.Remove("scp");
+    JsonWebTokenHandler.DefaultInboundClaimTypeMap.Add("scp", "scope");
 
 `Issue 446 <https://github.com/ThreeMammals/Ocelot/issues/446>`_ contains some code and examples that might help with Okta integration.

```

Comment:
1. I believe line 187, referencing issue #446 can be removed due to its outdated nature, correct? Should we always remap claims for Okta?
2. What about utilizing `Clear()` method as in the [MS Learn documentation](https://learn.microsoft.com/en-us/aspnet/core/security/authentication/claims?view=aspnet-8.0#claims-namespaces-namespaces)?